scripts/helm/apollo-portal/Chart.yaml

```diff
@@ -2,11 +2,11 @@ apiVersion: v2
 name: apollo-portal
 description: A Helm chart for Apollo Portal
 type: application
-version: 0.1.0
+version: 0.1.1
```

Comment:
Please change the version to 0.2.0 to align with the version in apollo-service, I think I forgot to change this version last time.

Reply:
modified